tools/topology/topology2/platform/intel/hw_config_simple.conf

```diff
@@ -0,0 +1,35 @@
+# Simple hardware config class definition for HDA/SDW/DMIC type DAIs. All attributes defined herein
+# are namespaced by alsatplg to "Object.Base.hw_config.1.attribute_name"
+#
+# Object.Base.hw_config.1 {
+# 		id		0
+#		name		"HDA0"
+# 	}
+#
+# where 1 is the unique instance ID for the hw_config object within the same alsaconf
+# node.
+
+Class.Base."hw_config" {
```

Comment:
I am not following how we can have the same class name for two different definitions, here and https://github.com/thesofproject/sof/pull/7370/commits/07ffe5d2e55e6f245e639eb6d2ecacfe5e47090b#diff-6097d83c177264e03795b3cb083915a0153a69fe445ea6046004160e75209628R15

Reply:
This is alsatplg compiler restriction for now. We can only include one or the other so we have only one class definition included. so the class name is the same "hw_config" because thats what the compiler expects